pkg/controller/mongodb/statefulset.go

```diff
@@ -35,9 +35,15 @@ spec:
   template:
     metadata:
       labels:
+        app.kubernetes.io/instance: common-mongo
         app: icp-mongodb
         release: mongodb
       annotations:
+        productID: "068a62892a1e4db39641342e592daa25"
+        productName: "IBM Cloud Platform Common Services"
+        productID: "068a62892a1e4db39641342e592daa25"
```

Comment:
There are two product ID. Should these annotations be added to the operator deployment as well?

Reply:
They just asked for the service I don't think they want the operator in it